### DISCUSSION TITLE
Parallel (next) runner combination with the VT tests and loader

### DISCUSSION THREAD

**Discussion by pevogam** [avocado-framework/avocado#4374] [Q&A]
Hi all, as Avocado VT is still very useful to many and everybody there could also very much benefit form some parallelism where multiple tasks could be managed by Avocado I wanted to ask do you think it is possible and for that matter meaningful to run VT tests through the new runner?

If I try to select a VT test with the parallel runner I am getting an incompatibility so I guess at the moment this is too much to ask for:
```
# avocado run io-github-autotest-qemu.shutdown --dry-run 
JOB ID     : 0000000000000000000000000000000000000000
JOB LOG    : /var/tmp/avocado-dry-run-qbrgrvli/job-2021-01-07T19.56-0000000/job.log
 (1/1) io-github-autotest-qemu.shutdown: CANCEL: Test cancelled due to --dry-run (0.00 s)
RESULTS    : PASS 0 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 1
JOB TIME   : 0.66 s
# avocado run io-github-autotest-qemu.shutdown --dry-run --test-runner='nrunner'
Could not resolve references: io-github-autotest-qemu.shutdown
```

But how about the general pros and cons of enabling this? Avocado VT had the job lock plugin which could prevent jobs from running in the same environment but this could also be turned off for cases where users (test runners) can make sure no data of the same vms images is accessed and thus no corruption happens. If the parallel runner would then use a simple process spawner would it be possible to manage multiple Avocado VT tasks at the same time?

**Comment by willianrampazzo** [comment]
@clebergnu showed us a work in progress at the end of 2020, right before he left for vacation. So, this is a work in progress and will be available in the near future.
